HackerRank/Plus Minus/main.js

```diff
@@ -0,0 +1,22 @@
+function plusMinus(arr) {
+  const parameterLength = arr.length;
+  const positive = [];
+  const zero = [];
+  const negative = [];
+  // loop for pushing the deferent values
+  for (let i = 0; i < parameterLength; i++) {
+    if (arr[i] > 0) {
+      positive.push(arr[i]);
+    } else if (arr[i] == 0) {
+      zero.push(arr[i]);
+    } else if (arr[i] < 0) {
+      negative.push(arr[i]);
+    }
+  }
+  // the math for proportion of values
+  console.log((positive.length / parameterLength).toFixed(6)),
```

Comment:
extra space at the begging like the lines after it

Reply:
this is from the Prettier - Code formatter when i save makes a space